compiler/rustc_hir/src/hir.rs

```diff
@@ -3456,7 +3456,7 @@ impl<'hir> Node<'hir> {
     /// ```ignore (illustrative)
     /// ctor
     ///     .ctor_hir_id()
-    ///     .and_then(|ctor_id| tcx.hir().find(tcx.hir().get_parent_node(ctor_id)))
+    ///     .and_then(|ctor_id| tcx.hir().find_parent_node(ctor_id))
```

Comment:
At a glance, this seems incorrect.

Reply:
Can you be more specific? 

`get_parent_node(_: HirId) -> HirId` was renamed to `parent_id(_: HirId) -> HirId`, and a *new* `find_parent_node(_: HirId) -> Optiion<Node>` was introduced.

Reply:
Of course :facepalm:

Reply:
Wait -- what modification to the PR title do you want? Also mentioning adding `{find,get}_parent`?